KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -74,7 +74,19 @@ Quorum is considered reached when at least half of the chairs (rounded up) are p
 
 Chairs of the KOC may abstain from a vote. Abstaining chairs will only be considered as contributing to quorum, in the event that a vote is called in a meeting.
 
+## Committee members
+
+Seats on the Outreach Committee are held by an individual, not by their employer.
+
+The current membership of the committee is (listed alphabetically by first name):
```

Comment:
Thank you for this @varodrig!
Can we also list KOC Leads here ?
I know that @chasecadet would like to nominate @tarekabouzeid as KOC lead.

Reply:
Sure, thanks for this!

Reply:
Thank you